src/main/kotlin/blackjack/model/PlayerFactory.kt

```diff
@@ -0,0 +1,9 @@
+package blackjack.model
+
+object PlayerFactory {
+    fun with(names: List<String>) = names.map { Player(it) }
+
+    fun createDealer() = Player(DEALER)
```

Comment:
```suggestion
    fun createPlayers(names: List<String>) = names.map { Player(it) }

    fun createDealer() = Player(DEALER)
```

I believe having consistent naming strategy allows us to maintain project more easily,
especially when working on it for a long time or with many teammates.

It makes code more predictable and less likely to make mistakes.
This is just my suggestion and I'm actually curious about your opinion :)

Reply:
I thought about creating the dealer together with `with(names)`, but due to poor design choices -- checking the dealer by reference -- the dealer and player ended up being created separately with the same class. 

It would have been better to change it to "create", but I also liked the name "with" so I left it for the future. I thought this could also be applied to names later. but I can say now, was't that good idea 😢

Reply:
I wouldn’t say it was a bad idea and honestly, my suggestion wasn’t a big one either.
To put it simply: naming things consistently just helps. That’s all 🙂

Reply:
I see. How to set up and accept language and words is important to me, so it's a part that I keep thinking about, which is why I wrote a bit lengthy. I'll try to think about it a bit more simply. Thank you.